[2021-05-07 15:24:08,065] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bash_dag.print_date 2020-05-02T19:08:00+00:00 [queued]>
[2021-05-07 15:24:09,370] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bash_dag.print_date 2020-05-02T19:08:00+00:00 [queued]>
[2021-05-07 15:24:09,371] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-05-07 15:24:09,372] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-05-07 15:24:09,372] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-05-07 15:24:09,638] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): print_date> on 2020-05-02T19:08:00+00:00
[2021-05-07 15:24:09,733] {standard_task_runner.py:52} INFO - Started process 5722 to run task
[2021-05-07 15:24:09,817] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'bash_dag', 'print_date', '2020-05-02T19:08:00+00:00', '--job-id', '25', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test_dag1.py', '--cfg-path', '/tmp/tmpw65q9w4x', '--error-file', '/tmp/tmpby2zccgp']
[2021-05-07 15:24:09,850] {standard_task_runner.py:77} INFO - Job 25: Subtask print_date
[2021-05-07 15:24:11,832] {logging_mixin.py:104} INFO - Running <TaskInstance: bash_dag.print_date 2020-05-02T19:08:00+00:00 [running]> on host jaarora-WX-1.corp.adobe.com
[2021-05-07 15:24:13,892] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=jacrod2901@gmail.com
AIRFLOW_CTX_DAG_OWNER=jacrod2901
AIRFLOW_CTX_DAG_ID=bash_dag
AIRFLOW_CTX_TASK_ID=print_date
AIRFLOW_CTX_EXECUTION_DATE=2020-05-02T19:08:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-05-02T19:08:00+00:00
[2021-05-07 15:24:13,893] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-05-07 15:24:13,897] {bash.py:158} INFO - Running command: date
[2021-05-07 15:24:14,011] {bash.py:169} INFO - Output:
[2021-05-07 15:24:14,050] {bash.py:173} INFO - Fri May  7 15:24:14 IST 2021
[2021-05-07 15:24:14,052] {bash.py:177} INFO - Command exited with return code 0
[2021-05-07 15:24:15,150] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=bash_dag, task_id=print_date, execution_date=20200502T190800, start_date=20210507T095408, end_date=20210507T095415
[2021-05-07 15:24:16,335] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-07 15:24:16,431] {local_task_job.py:146} INFO - Task exited with return code 0
